app/assets/stylesheets/components/_cards-grid.scss

```diff
@@ -41,20 +41,20 @@
 // Medium devices (tablets, 768px and up)
 @media (min-width: 768px) {
   .cards {
-    grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr;
+    grid-template-columns: 1fr 1fr
```

Comment:
As you don't use the grid for responsiveness but the `.phone` class. I think you can delete all the media queries in this file :)